great_expectations/util.py

```diff
@@ -152,9 +152,27 @@ def profile_function_call(*args, **kwargs) -> Any:
 def measure_execution_time(
     execution_time_holder_object_reference_name: str = "execution_time_holder",
     execution_time_property_name: str = "execution_time",
+    method: str = "process_time",
```

Comment:
Instead of a `str` can we pass in a `Callable[None, float]` here? This could default to `time.process_time`.

Reply:
@billdirks Please feel free to try that -- whichever works best would be fine with me as long as it works inside the code and in the notebook.  Thanks!